### PR TITLE
LPS-120852 Add test to resolve generated projects with the distro jar

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
@gamerson, CI doesnt run this test, so there's nothing else to trigger.

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/platform.bndrun [hunk 1]
```diff
@@ -0,0 +1,50 @@
+##
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/platform.bndrun:1]: @rotty3000 FYI: this file is going into workspace template now.  Thanks for the file.  Let us know if you have any suggested changes.

**Review by gamerson — APPROVED** [review]

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/petershin/liferay-portal/pull/1190. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**